Project-I-Social-Card/social-card/src/components/HeaderComponents/Header.css

```diff
@@ -1 +1,72 @@
 /* All styles for your header Components live here */
+
+.lambda-logo {
+    width: 85px;
+    height: 85px;
+    
+}
+
+.header-container {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    flex-wrap: wrap;
+   
+}
+
+.banner {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
```

Comment:
You could also set `border-radius` for the banner image to match the design file more closely.